recipes/ARIA-tools/meta.yaml

```
@@ -0,0 +1,87 @@
+{% set name = "ARIA-tools" %}
+{% set version = "1.1.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/aria-tools/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: c544fee1c539f38e23897d27332a6af0f45d48874e651a096993dbd311cbbf48
+
+build:
+  script:
+    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+    - {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [py<38 or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - pip
+    - pybind11 >=2.5.0
+    - setuptools >61.0
+    - setuptools_scm >6.2
+    - python-build
+    - numpy
+    - cython
+  run:
+    - python
+    - asf_search
+    - cartopy
+    - gdal >=3.4.1
+    - h5py
+    - joblib
+    - matplotlib-base
+    - netcdf4
+    - pandas
+    - xarray
+    - rasterio
+    - rioxarray
+    - pip
+    - pyproj
+    - requests
+    - scipy <1.10
+    - mintpy
+    - shapely
+
+test:
+  imports:
+    - ARIA-tools
```

```suggestion
    - ARIAtools
```
I believe this is the issue -- the actual python package installed is `ARIAtools`